macros/edr/data_monitoring/monitors_query/dimension_monitoring_query.sql

```diff
@@ -46,7 +46,7 @@
                     dimension_value,
                     1 as joiner
                 from filtered_monitored_table
-            )
+            ) rs
```

Comment:
What are those?

Reply:
In postgres subqueries in FROM must have an alias

Reply:
I see. What does `rs` stand for?

Reply:
result set. I use it in my queries